tor/add_onion.go

```diff
@@ -22,6 +26,12 @@ const (
 
 	// V3 denotes that the onion service is V3.
 	V3
+
+	// key parameter that Tor accepts for a new V2 service.
+	V2keyParam string = "RSA1024"
+
+	// key parameter that Tor accepts for a new V3 service.
+	V3keyParam string = "ED25519-V3"
```

Comment:
```suggestion
	v3KeyParam = "ED25519-V3"
```

Note the lowercase `v`, the constant doesn't need to be exported.